layers/+completion/helm/funcs.el

```diff
@@ -513,6 +513,23 @@ Removes the automatic guessing of the initial value based on thing at point. "
     (set-text-properties 0 (length input) nil input)
     (helm-find-files-1 input)))
 
+;; ido-mode remaps some commands to ido counterparts.  We want default Emacs key
+;; bindings (those under C-x) to use ido, but we want to use the original
+;; commands in Spacemacs key bindings (those under M-m or SPC) so that they use
+;; `read-file-name-function', `completing-read-function',
+;; `completion-in-region-function', etc. configured by Helm or Ivy etc.  The
+;; following aliases allow us to bind Spacemacs keys to the original commands.
+
+(defalias 'spacemacs/find-file-other-frame 'find-find-other-frame)
+
+(defalias 'spacemacs/dired-other-frame 'dired-other-frame)
+
+(defalias 'spacemacs/switch-to-buffer-other-frame 'switch-to-buffer-other-frame)
+
+(defalias 'spacemacs/insert-files 'insert-file)
```

Comment:
`spacemacs/insert-files` --> `spacemacs/insert-file`

Reply:
My reasoning for naming it `spacemacs/insert-files` was that helm allows selecting multiple files.  I did find that helm only really inserts one file, but that looks like a defect in helm (I'll look into fixing it upstream later).